orttraining/orttraining/python/training/ortmodule/_inference_manager.py

```diff
@@ -49,10 +46,6 @@ def execution_session_run_forward(execution_session, onnx_model, device, *inputs
             forward_output._ortvalue) for forward_output in forward_outputs)
         state = None
 
-        # Assert that the outputs and model device match
-        _utils._check_same_device(
```

Comment:
The previous call `_check_same_device` for input was moved, but this one seems to be removed. Is it intentional?

Reply:
Yes. This is intentional since ORT will not automatically change the device for outputs. This is consistent with what is currently happening in the training_manager